src/components/books/BookLayout.module.scss

```diff
@@ -23,42 +23,56 @@
 
 .searchHeader {
   display: flex;
+  flex-direction: column;
   align-items: center;
   justify-content: space-between;
+  gap: 2rem;
   margin-bottom: 3rem;
 }
 
-.searchInput {
-  display: none;
-  padding: 0.3rem 1rem;
-  border: 1px solid #ddd;
-  border-radius: 24px;
-  width: 300px;
+.searchTitle {
+  font-size: 1.5rem;
+  text-align: center;
+  text-decoration: underline;
+  text-decoration-color: $primary-color;
+  text-decoration-thickness: 0.4rem;
+  text-decoration-skip-ink: none;
+  text-underline-offset: -4px;
 
-  &:focus {
-    outline: none;
+  @media (min-width: $screen-sm-min) {
+    font-size: 2rem;
   }
+}
+
+.searchContainer {
+  position: relative;
+  width: 100%;
+  max-width: 600px;
 
-  @media (min-width: $screen-md-min) {
-    display: block;
+  @media (min-width: $screen-sm-min) {
+    width: 80%;
   }
 }
 
-.searchIcon {
-  display: flex;
-  justify-content: center;
-  position: relative;
-  background-color: transparent;
-  cursor: pointer;
-  border: none;
+.searchInput {
+  width: 100%;
+  padding: 0.4rem 2rem 0.4rem 1rem;
+  border: 2px solid $secondary-color;
+  border-radius: 40px;
+
+  &:focus {
+    outline: none;
```

Comment:
Ok, you have removed the outline on focus. So accessibility is killed. Let's now use the bit better `focus-visible` and ADD some alternative - border color change, something.

Even the outline maybe! I understand that focus trigger on mouse click as well, `focus-visible` does not! :)) maybe it will solve it